src/model/AppModel.as

```diff
@@ -0,0 +1,12 @@
+package model {
+[Bindable]
+public class AppModel {
+    public var showLogs:Boolean = true;
```

Comment:
OHHH I see. you're using it to enable/disable the log view. That's not so useful.

The checkbox should call a method (which does not exist in the ANE yet, as I now see) that enables/disables verbose logging (i.e.: to `adb` or the iOS device logs). https://github.com/Heyzap/unity-example-app/blob/master/Assets/Scripts/AdManager.cs#L189 is an example. There is a ticket to include this in the ANE, so for now you can comment this out of the test app.